src/Sylius/Bundle/ProductBundle/Controller/AssociationTypeController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ProductBundle\Controller;
+
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Association type controller.
+ *
+ * @author Leszek Prabucki <leszek.prabucki@gmail.com>
+ */
+class AssociationTypeController extends ResourceController
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function createAction(Request $request)
```

Comment:
I don't like this either, but here you have comment why: https://github.com/Sylius/Sylius/pull/2098#discussion_r19899796